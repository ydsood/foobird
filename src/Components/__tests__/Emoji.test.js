import React from "react";
import ReactDOM from "react-dom";
import renderer from 'react-test-renderer';
import Emoji from "../Emoji";

describe("Emoji", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Emoji />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders with emoji", () => {
      const emoji = renderer.create(<Emoji symbol="ðŸ’©" label="doo"/>).toJSON();
      expect(emoji).toMatchSnapshot();
  })
});
