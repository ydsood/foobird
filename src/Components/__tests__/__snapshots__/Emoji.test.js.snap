// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Emoji renders with emoji 1`] = `
<span
  aria-hidden="false"
  aria-label="doo"
  className="emoji"
  role="img"
>
  ðŸ’©
</span>
`;
